cyhy_report/customer/generate_report.py

```diff
@@ -1177,6 +1179,36 @@ def __determine_bubble_sizes(self, severities, vuln_counts):
             bubble_sizes.append(2 * vulns_ranked[severity] + 10)
         return bubble_sizes
 
+    def __figure_kev_severity_by_prominence(self):
+        severities = [i.lower() for i in reversed(SEVERITY_LEVELS[1:])]
```

Comment:
Are we still good to ignore `Informational` severity items? I know this is done elsewhere but it has a very "magic number" vibe since there's no explanation for why it's being done.

Reply:
We do not ingest (i.e. ignore) any vulnscan reports with severity 0 ("Informational"), therefore we do not have any vulnscan tickets with severity 0.  I can add a comment about that here if you like, but it's something that applies throughout this code.

Reply:
If you're inclined something here
https://github.com/cisagov/cyhy-reports/blob/4d255bd4ef7b48c564bfb8cc61086e5b69f09e67/cyhy_report/customer/generate_report.py#L82
would at least provide a central point of reference.

Reply:
Fair enough- I added a hopefully-helpful comment in https://github.com/cisagov/cyhy-reports/pull/71/commits/6ef2a68c7352afbc3ff68af1ec0f90dbae7d4ae1.

Reply:
Helpful enough to future readers ^^